apps/database/lib/database/schema/mod.ex

```diff
@@ -14,4 +15,11 @@ defmodule Database.Schema.Mod do
     has_many(:image, Database.Schema.ModImage)
     many_to_many(:tags, Database.Schema.ModTag, join_through: "mods_tags", unique: true)
   end
+
+  def changeset(record, params \\ :empty) do
```

Comment:
dialyzer complains about no local return here

Reply:
Database.Schema.Mod.changeset(_any(), :empty)

will never return since it differs in arguments with
positions 2 from the success typing arguments:

(
  {map(), map()}
  | %{:__struct__ => atom() | %{:__changeset__ => map(), _ => _}, atom() => _},
  :invalid | %{:__struct__ => none(), (atom() | binary()) => _}
)